### PR TITLE
Add list command test cases and fix some bugs 

### PR DESCRIPTION
**Pull request by wjacobw**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public EditCommand parse(String args) throws ParseException {
             throw new ParseException(EditCommand.MESSAGE_NOT_EDITED);
         }
 
-        return new EditCommand(index, editPersonDescriptor);
+        return new EditCommand(index, editPersonDescriptor); // more args here
```
Comment (xxiaoweii) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java:67]: i think this commit can be for another PR since it is not related to the list test cases

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -15,5 +15,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_TUTORIAL = new Prefix("--tutorial");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_ADD = new Prefix("--add");
+    public static final Prefix PREFIX_DELETE = new Prefix("--delete");
```
Comment (xxiaoweii) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:19]: Same for this changes

**Review by xxiaoweii — APPROVED** [review]
The added testcases lgtm!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -135,6 +156,10 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
         assertCommandSuccess(command, actualModel, expectedCommandResult, expectedModel);
     }
 
+    public static void asserCommandSuccess() {
```
Comment (frrrrry) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:159]: is this method necessary, because its just an empty method? also the method name has spelling error
